apps/shared/iccmaker.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 Yuan Tong. All rights reserved.
+// SPDX-License-Identifier: BSD-2-Clause
+
+#ifndef LIBAVIF_APPS_SHARED_ICCMAKER_H
+#define LIBAVIF_APPS_SHARED_ICCMAKER_H
+
+#include "avif/avif.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
Add a note to libavif maintainers that these ICC maker functions should stay here and shall not be moved to libavif itself (avif.h). We want to keep libavif itself out of the ICC generation game.

```
// Note to libavif maintainers: These ICC maker functions should stay here ...
```